src/extension.ts

```diff
@@ -57,12 +57,13 @@ async function openWindowsTerminal(profile: IWTProfile, uri?: vscode.Uri) {
     if (uri.authority.startsWith('ssh-remote+')) {
       // Handle SSH remotes first
       const host = uri.authority.split('+')[1];
-      const isWindows = !!uri.path.match(/^\/[a-z]:\//);
+      const isWindows = !!uri.path.match(/^\/[a-zA-Z]:\/.*$/);
+      const remoteMachine = await resolveSSHHostName(host);
```

Comment:
👍 can you split this out into a separate PR? The uri thing might take a bit to get right.